apps/cowswap-frontend/src/modules/usdAmount/apis/getCoingeckoUsdPrice.ts

```diff
@@ -18,7 +18,7 @@ export const COINGECK_PLATFORMS: Record<SupportedChainId, string | null> = {
   [SupportedChainId.SEPOLIA]: null,
 }
 
-const BASE_URL = 'https://api.coingecko.com/api/v3/simple/token_price'
+const BASE_URL = 'https://cow-web-services.vercel.app/api/serverless/coingeckoProxy'
```

Comment:
don't we need a DNS for this?

Reply:
Sure!  but now is not needed, right? we plan to change this to BFF instead of using vercel

Reply:
Let's update that once BFF is live.
This can be used as is in the mean time.

For BFF we still need to check/configure the DNS caching, otherwise we'd be back at hitting coingecko too hard.